### ISSUE TITLE
Ability to colorize imports?

### ISSUE THREAD

**Issue by Syntaf** [MagicStack/MagicPython#223]
* **Editor name and version**: VSCode 1.49.0
* **Platform**: Windows 10
* **Color scheme**: Dark+
* **MagicPython version**: Installed via Python extension v2020.8.108011
* **A sreenshot**: 
![image](https://user-images.githubusercontent.com/1827631/93114728-cbd8b080-f66f-11ea-9cf1-0721c9b1689b.png)

* **5-10 lines of surrounding code**:

```python
from django.utils import timezone
from django.views import generic

from .models import Question, Choice

class DetailView(generic.DetailView):
    model = Question
    template_name = 'polls/detail.html'

class ResultsView(generic.DetailView):
    model = Question
    template_name = 'polls/results.html
```

Just as the above code snippet does, is it possible to colorize class imports in VSCode?

I've tried searching the web for an answer on this topic but couldn't find one; I'm submitting an issue here in hopes of someone either telling me that this is _intended_ but not working, or that this is not supported.

Thanks for any info!

**Comment by 1st1** [comment]
> Just as the above code snippet does, is it possible to colorize class imports in VSCode?

No, not really, because syntax highlighter doesn't know if it's a class or just a weirdly named module.

@vpetrovykh Victor, it's weird that GH highlights that though. Maybe they no longer use MP?

**Comment by elprans** [comment]
> Maybe they no longer use MP?

Yeah, it appears they no longer use https://github.com/github/linguist to highlight the source.

**Comment by 1st1** [comment]
That explains the bugs, sigh.

**Comment by Syntaf** [comment]
> > Just as the above code snippet does, is it possible to colorize class imports in VSCode?
> 
> No, not really, because syntax highlighter doesn't know if it's a class or just a weirdly named module.
> 
> @vpetrovykh Victor, it's weird that GH highlights that though. Maybe they no longer use MP?

Gotcha, thanks for the explanation.

In my project the module naming is fairly consistent where I can guarantee that:
- A lower case import is a function `from x import foo`
- A camel case import is a class `from x import Foo`

Is it possible for me to manually colorize these within my theme using the VSCode `editor.tokenColorCustomizations` setting?

Thanks!

**Comment by overjoy** [comment]
So what's magic about this project if it can't distinguish imports, variables and classes?
Here is the comparison with PTVS:

<img width="272" alt="image" src="https://user-images.githubusercontent.com/510509/131321222-d886b49b-7b26-4b47-8964-639f85479b8b.png">

<img width="425" alt="image" src="https://user-images.githubusercontent.com/510509/131321315-77d45cd4-c324-4b17-af67-d5748b999980.png">
